cli/integration_tests/composable_config/composing-override-values-deps.t

```diff
@@ -6,39 +6,45 @@ Setup
 # The workspace does not have a turbo.json config. This test checks that both regular dependencies
 # and Topological dependencies are retained from the root config.
 
-# 1. First run, assert that dependet tasks run `dependsOn`
-  $ ${TURBO} run override-values-task-with-deps --filter=override-values > tmp.log
-# Validate in pieces. `omit-key` task has two dependsOn values, and those tasks
-# can run in non-deterministic order. So we need to validate the logs in the pieces.
-  $ cat tmp.log | grep "in scope" -A 2
+# Run override-values-task-with-deps. In the root turbo.json it has two dependsOn values
+# but in the workspace, we override to dependsOn: []. This test validates that only the
+# top level task "override-values-task-with-deps" should run. None of the dependencies should run.
+  $ ${TURBO} run override-values-task-with-deps --filter=override-values
```

Comment:
I guess this test was just written wrong... maybe the result of a large PR where I was `-i` often.